presentation/common/src/presentation-common/content/Descriptor.ts

```diff
@@ -206,8 +211,18 @@ export interface DescriptorOverrides {
     direction: SortDirection;
   };
 
-  /** [ECExpression]($docs/presentation/advanced/ECExpressions.md) for filtering content */
+  /**
+   * [ECExpression]($docs/presentation/advanced/ECExpressions.md) for filtering content
+   * @deprecated Use fieldsFilterExpression instead.
+   */
   filterExpression?: string;
+  /** [ECExpression]($docs/presentation/advanced/ECExpressions.md) for filtering content */
+  fieldsFilterExpression?: string;
+  /**
+   * Content instances filter
+   * @alpha
+   */
+  instanceFilter?: InstanceFilterDefinition;
```

Comment:
I think the docs between these two should be cross-linked, explaining the differences between them. 

Of course `instanceFilter` can't be referenced at the moment due to being `@alpha`, so for the time being only `fieldsFilterExpression` should be referenced from the docs of `instanceFilter`.

The attributes are repeated in several places, so maybe adding a "Content filtering" doc page, explaining all the different ways to filter content, including these attributes, would be the most convenient way to document this.